### ISSUE TITLE
import .ics files

### ISSUE THREAD

**Issue by geier** [geier/khal#111]
import complete calendar .ics files

this could also be implemented in vdirsyncer, what do you think @untitaker ?

blocked by #118

**Comment by untitaker** [comment]
I think maybe such things should be implemented in a ``vdir-utils`` package, which would be a set of standalone CLI tools.

For this particular feature, a command with an interface similar to coreutils' ``split`` could be provided:

    vobj-split my-file.ics ~/.calendars/default/

The main advantage here is that it is neither specific to khal, vdirsyncer or even vdir, the interface is easy to use, regardless of whether you already use khal/vdirsyncer or not.

**Comment by geier** [comment]
When implementing (parts of) RFC 5546 this will be more or less a by
product (and I think RFC 5546 is a better fit for khal, because one
probably wants to view the event(s) and the event's date before
accepting or declining.

Since implementing RFC 5546 is a lot of work, it probably makes sense to
implement this independently now.

Quoting Markus Unterwaditzer (2015-02-02 14:36:54)

**Comment by geier** [comment]
this will hopefully come together with the new database layout, I'm
working on that in `rrrule_exceptions` (sadly not enough time atm)

Quoting James Mills (2014-11-26 16:19:16)

**Comment by untitaker** [comment]
@geier It seems that your replies appear to be sent very delayed, also in #130 and #131.

**Comment by geier** [comment]
ran msmtp-runqueue on a seldomly used laptop...

**Comment by bepolymathe** [comment]
Hi... any news about this. Pipe file .ics like this don't work
`
| khal import
`
Any idea ?

**Comment by geier** [comment]
@bepolymathe please open a new issue for this, thank you!
